tools/validators/instance_validator/tests/fake_instances/GOOD/new_format_singleton.yaml

```diff
@@ -15,7 +15,7 @@
 9a86a19b-b687-4db1-888e-2cf34d04b74c:
   type: HVAC/CHWS_WDT
   code: CHWS-1
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.